pyxrf/xanes_maps/xanes_maps_api.py

```diff
@@ -762,6 +757,12 @@ def _fit_xanes_map(map_data, absorption_refs):
     for ny in range(nny):
         for nx in range(nnx):
             result, residual = nnls_fit(map_data[:, ny, nx], absorption_refs, weights=None)
+
```

Comment:
Yes, I checked again, you are right. But for some reason this is also older code (changed yesterday afternoon). They requested fitting error to be represented as R-factor instead of MSE, so I modified the code to compute R-factor and returned residual is just ignored.